### PR TITLE
Photos: title and menus in the cozy bar

### PR DESCRIPTION
**Pull request by y-lohse**
Don't merge until we have https://github.com/cozy/cozy-client/pull/116 and a new version of cozy-client!

- I moved the title of the pages to the cozy-bar on mobile view, that was relatively easy.
- I also moved all the menus toolbar there
- The toolbar menus now use cozy-ui menu's
- Once renderer inside the cozy-bar, the components get the cozy-bar contact, so any HOC that relies on context won't work. To workaround this, I had to pass the Toolbars instance of `t`, `router`, and move some sharing code around

### CODE REVIEW

**Comment by y-lohse** [comment]
[(1) 🐛 Afficher le nom de la page et le menu "more" sur Photos (mobile)](https://trello.com/c/2qfkvO7E/1062-1-%F0%9F%90%9B-afficher-le-nom-de-la-page-et-le-menu-more-sur-photos-mobile)

File: src/photos/ducks/albums/components/AlbumsView.jsx [hunk 1]
```diff
@@ -20,19 +21,21 @@ const Content = ({ list }) => {
...
+  render({ t }) {
     if (this.props.children) return this.props.children
     if (!this.props.albums) {
       return null
     }
     return (
       <div className={styles['pho-content-wrapper']}>
         <Topbar viewName="albums">
-          <AlbumsToolbar />
+          <AlbumsToolbar t={t} />
```
Comment (goldoraf) [inline src/photos/ducks/albums/components/AlbumsView.jsx]: Why pass `t` here and not using `translate` HOC for the `AlbumToolbar` too? Am I missing something?

File: src/photos/ducks/albums/components/AlbumsView.jsx [hunk 2]
```diff
@@ -20,19 +21,21 @@ const Content = ({ list }) => {
   }
 }
 
-export default class AlbumsView extends Component {
-  render() {
+class AlbumsView extends Component {
+  render({ t }) {
```
Comment (goldoraf) [inline src/photos/ducks/albums/components/AlbumsView.jsx]: This way of getting props being specific to Preact, we shouldn't add more but preferably remove what we may find, should we?

File: src/photos/ducks/albums/components/AlbumsView.jsx (hunk 1 above)
Comment (y-lohse) [reply src/photos/ducks/albums/components/AlbumsView.jsx]: Yeah but it's easy to miss: The `Topbar` may wrap the `AlbumsToolbar` inside the cozy-bar. If that happens, `t` injected via the HOC will have the bar's translation context, so we need to pass it an instance of `t` that is bound to the correct context.

**Pull request merged**